subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -10,5 +10,5 @@ scala-xml=1.2.0
 scala=2.12
 scalatest=3.0.7
 slf4j=1.7.26
-spock=1.2-groovy-2.5
+spock=1.3-groovy-2.5
```

Comment:
This file shouldn't be changed manually, but via the `updateInitPluginTemplateFile`. In order to update the Spock version, I guess this line needs to be changed: https://github.com/gradle/gradle/blob/8ab371109b8d75117edcb1fa93a3bf92ee764dad/subprojects/build-init/build-init.gradle.kts#L90

Reply:
Thanks for the review, Stefan!
I've ran `gw updateInitPluginTemplateVersionFile` for https://github.com/gradle/gradle/pull/9486/files#diff-10559753926ae8cd59f961f3aca772c3R90 and don't see any changes locally. Am I missing something?

Reply:
Sorry, looks like I missed the updated line above: https://github.com/gradle/gradle/pull/9486/files#diff-10559753926ae8cd59f961f3aca772c3R90